### PR TITLE
Ks 2022 07 issues

### PR DESCRIPTION
**Pull request by Rineee**
Needs https://github.com/liqd/adhocracy-plus/pull/1990

### CODE REVIEW

File: containers/Ideas/Phase.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {View} from 'react-native';
+
+import {TextSourceSans} from '../../components/TextSourceSans';
+import { styles } from './Phase.styles';
+
+export const Phase = props => {
+  const {activePhase, futurePhases, pastPhases} = props;
+  const startDate = (futurePhases) ? futurePhases[0].start_date : null;
```
Comment (goapunk) [inline containers/Ideas/Phase.js:9]: are the parentheses needed here and in the line below?

File: containers/Ideas/IdeaProject.js [hunk 2]
```diff
@@ -146,24 +139,11 @@ export const IdeaProject = (props) => {
...
-                      {activePhase.description}
-                    </TextSourceSans>
-                  </View>
-                ) : (
-                  <View style={styles.phaseContainer}>
-                    <TextSourceSans>
-                      No active phase found.
-                    </TextSourceSans>
-                  </View>
+                  <Phase
```
Comment (fuzzylogic2000) [inline containers/Ideas/IdeaProject.js:142]: Cool! An extra phase component is a good idea!

File: containers/Ideas/Phase.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  const endDate = (pastPhases) ? pastPhases[pastPhases.length-1].end_date : null;
+
+  return (
+    activePhase ?  (
```
Comment (fuzzylogic2000) [inline containers/Ideas/Phase.js:13]: Yeah, all the `?` and `:` is hard to understand, but looking at it for a while it makes sense. :woman_shrugging: I am fine with leaving it like that.

File: containers/Ideas/Phase.js (hunk 1 above)
Comment (Rineee) [reply containers/Ideas/Phase.js]: I thought yes, but that was wrong :sweat_smile: So removed them now, thanks!

**Pull request merged**